### ISSUE TITLE
Executing a Script Without Parameters Using the Shortcut for Execute with Parameters Produces Error

### ISSUE THREAD

**Issue by MrayerMSFT** [microsoft/poshtools#688]
As title states, this box will appear:

![ref](https://cloud.githubusercontent.com/assets/12631548/9092324/63de5bf4-3b5a-11e5-8a0e-544775602a02.PNG)

**Comment by MrayerMSFT** [comment]
Investigated: null exception occurs at line 47 of PowerShellParseUtilities:

``` C#
var parametersList = paramBlockAst.Parameters.
                Where(p => !DataTypeConstants.UnsupportedDataTypes.Contains(p.StaticType.FullName)).
                ToList();
```

because paramBlockAst is null. Should be an easy fix depending on what we want the user experience to be.

**Comment by MrayerMSFT** [comment]
Fixed by adamdriscoll/poshtools/#305
